tools/e2etesting/GetSecrets.ps1

```diff
@@ -16,4 +28,25 @@ foreach ($secret in $secrets) {
 	$values[$secret.Name.ToUpperInvariant().Replace("-", "_")] = $secretValueText
 }
 
-$values | ConvertTo-Json -Depth 10
\ No newline at end of file
+# Find the full path of the launchSettings.json file
+$folderName = "e2e-tests"
+$currentPath = (Get-Location).Path
+$path = Get-ChildItem -Path $currentPath $folderName -Recurse 
+while (!$path) {	
+    $currentPath = $currentPath + '\..'
+    $path = Get-ChildItem -Path $currentPath $folderName 
+}
+$settingsFile = $path.FullName + "\IIoTPlatform-E2E-Tests\Properties\launchSettings.json"
```

Comment:
@MaidaB, would you please consider making the script cross-platform so that it can run on Linux/WSL as well ? For that you would only need to use PowerShell functions such as `Join-Path` instead of using Windows path separator. Line `36` would also need to change slightly. Please feel free to ignore this comment if this is not a priority.

Reply:
I tested it on a Linux machine and it works fine when PowerShell is installed